packages/cli-core/API.md

```diff
@@ -62,7 +57,7 @@ export class Printer {
     constructor(minimumLogLevel: LogLevel, stdout?: NodeJS.WriteStream, stderr?: NodeJS.WriteStream, refreshRate?: number);
     indicateProgress(message: string, callback: () => Promise<void>): Promise<void>;
     log(message: string, level?: LogLevel, eol?: boolean): void;
-    print: (message: string, colorName?: COLOR) => void;
+    print: (message: string | string[], formatName?: keyof typeof format) => void;

```

Comment:
this method signature should be `print: (message: string) => void`. The caller can call format methods and pass the result to print. Here's a usage example: https://github.com/aws-amplify/amplify-backend/blob/26cdffd3a9960a610bf5315f83941afea0596f89/packages/create-amplify/src/amplify_project_creator.ts#L57-L58

Reply:
It could, but not accurate. If `print.list` only accept `string[]`, we should take it into consideration.

Reply:
that would be `printer.print(format.list(elements))`. The format method returns a single string.

Reply:
ohhhh, I see what you meant. Done 8cac3740d7